indico/core/db/sqlalchemy/util/models.py

```diff
@@ -142,6 +142,18 @@ def populate_from_dict(self, data):
                 raise ValueError("{} has no attribute '{}'".format(cls.__name__, key))
             setattr(self, key, value)
 
+    def populate_from_attrs(self, obj, attrs):
```

Comment:
Would fit too but to me that name sounds more like something copying all attributes